<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerPurchase</class>
 <widget class="QDialog" name="CustomerPurchase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Customer ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="8">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Item Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Complete Purchase</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Refresh Inventory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
